Fixed #29356 -- Clarified docs for QueryDict.getlist() default. (#13510)
django/django: docs/ref/request-response.txt

.. method:: QueryDict.getlist(key, default=None)

    Returns a list of the data with the requested key. Returns an empty list if

>     the key doesn't exist and a default value wasn't provided (i.e., is None).
>     It's guaranteed to return a list unless the default value provided isn't a list.

Reviewer: Slight edit, plus wrap to 79 chars: 

```suggestion
    the key doesn't exist and a default value wasn't provided, or was ``None``.
    It's guaranteed to return a list unless the default value provided isn't a
    list.
```
